### ISSUE TITLE
Add ability to forget old chats / auto-forget old chats

### ISSUE THREAD

**Issue by FantasyCookie17** [Nheko-Reborn/nheko#244]
Apparently your Matrix server keeps a list of old rooms you were in until you decide to forget them, which can be done in Element and Riot, but not in Nheko. Adding support for performing this action, or even auto-forgetting (would be way less annoying tbh) would be great.

**Comment by LorenDB** [comment]
I don't think that we even show those rooms, so I think this could be closed.

**Comment by deepbluev7** [comment]
No, there is an API to tell the server, that you don't need the room anymore and Nheko has no way to call that.

**Comment by LorenDB** [comment]
I understand that, but as far as I can tell you can't see the left rooms in nheko, so I don't think we need to worry about forgetting them. Or am I missing something?

**Comment by deepbluev7** [comment]
Yes, you are forgetting that the server can't gc them :3
